tests/transport/aio/test_aiohttp.py

```diff
@@ -13,4 +13,48 @@
 # limitations under the License.
 
 import google.auth.aio.transport.aiohttp as auth_aiohttp
-import pytest # type: ignore
+import pytest  # type: ignore
+import pytest_asyncio
+import asyncio
+from google.auth import exceptions
+from aioresponses import aioresponses
+
+# TODO (ohmayr): Verify if we want to optionally run these test cases
+# if aohttp is installed instead of raising an exception.
```

Comment:
I think it makes sense to raise the exception so these aren't silently skipped.

Reply:
Okay makes sense. I've kept it as is and removed the TODO.